test/core/xds/xds_client_test.cc

```diff
@@ -930,19 +964,12 @@ TEST_F(XdsClientTest, BasicWatch) {
   EXPECT_EQ(resource->name, "foo1");
   EXPECT_EQ(resource->value, 6);
   // Check metric data.
-  EXPECT_THAT(metrics_reporter_->resource_updates_valid(),
-              ::testing::ElementsAre(::testing::Pair(
-                  ::testing::Pair(kDefaultXdsServerUrl,
-                                  XdsFooResourceType::Get()->type_url()),
-                  1)));
-  EXPECT_THAT(metrics_reporter_->resource_updates_invalid(),
-              ::testing::ElementsAre());
-  EXPECT_THAT(
-      GetResourceCounts(),
```

Comment:
Looks like you accidentally removed this check.

Reply:
Done